Samples.xcodeproj/project.pbxproj

```diff
@@ -57,6 +57,7 @@
 		00E5401C27F3CCA200CF66D5 /* SamplesApp.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00E5400C27F3CCA100CF66D5 /* SamplesApp.swift */; };
 		00E5401E27F3CCA200CF66D5 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00E5400D27F3CCA100CF66D5 /* ContentView.swift */; };
 		00E5402027F3CCA200CF66D5 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 00E5400E27F3CCA200CF66D5 /* Assets.xcassets */; };
+		75E92C6129CCBCF900111226 /* NavigateARouteView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = E008221428760157002AD138 /* NavigateARouteView.swift */; };
```

Comment:
Sorry for the late notice. Can you help rename the sample to "Navigate route" according to the common design? There was an effort to standardize the names across platforms, and the old iOS sample name deviates from the common design. Thanks.

- the folder name
- README title
- title in metadata
- swift filename
- view type name (NavigateARouteView -> NavigateRouteView)